scenarioo-client/app/shared/utils/urlContextExtractor.service.spec.ts

```diff
@@ -19,53 +19,53 @@ import {UrlContextExtractorService} from './urlContextExtractor.service';
 
 declare var angular: angular.IAngularStatic;
 
-describe('UrlContextExtractorService', () => {
+fdescribe('UrlContextExtractorService', () => {
```

Comment:
Was there a special reason to use fdescribe instead of describe? If so a comment would be helpful.

Reply:
there is never a good reason for this - it is not allowed to commit or fdescribes! ;-)